USAGE.md

```diff
@@ -1,370 +1,707 @@
+| `x_mitre_aliases` | string[] | List of aliases for the given software |
+
+
+### Relationships
+
+Objects in ATT&CK are related to each other via STIX [relationship](https://docs.oasis-open.org/cti/stix/v2.0/csprd01/part2-stix-objects/stix-v2.0-csprd01-part2-stix-objects.html#_Toc476230970) objects. These relationships convey concepts like groups using techniques (also called "procedure examples" on the technique pages), the hierarchy of techniques and sub-techniques, and so on. 
 
-## Python Library
-To begin querying STIX 2.0 data, you must first have a [DataSource](http://stix2.readthedocs.io/en/latest/guide/datastore.html). For these examples, we will simply use a [FileSystemSource](http://stix2.readthedocs.io/en/latest/guide/filesystem.html). The ATT&CK corpus must first be cloned or downloaded from [GitHub](https://github.com/mitre/cti).
+Relationships oftentimes have descriptions which contextualize the relationship between the objects.
 
-**Note** for this example we used the enterprise-attack corpus to demonstrate usage. The same can be done for all other data sources.
+| Source Type | Relationship Type | Target Type | Custom Type? | About |
+|:------------|:------------------|:------------|:----|:------|
+| `intrusion-set` | `uses`        | `malware` or `tool` | No | Group using a software. |
+| `intrusion-set` | `uses`        | `attack-pattern`    | No | Group using a technique, which is also considered a procedure example. |
+| `malware` or `tool` | `uses`    | `attack-pattern`    | No | Software using a technique, which is also considered a procedure example. |
+| `course-of-action`  | `mitigates` | `attack-pattern`  | No | Mitigation mitigating technique. |
+| `attack-pattern`    | `subtechnique-of` | `attack-pattern` | Yes | Sub-technique of a technique, where the `source_ref` is the sub-technique and the `target_ref` is the parent technique. |
+| any type    | `revoked-by`      | any type | Yes | The target object is a replacement for the source object. Only occurs where the objects are of the same type, and the source object will have the property `x_mitre_revoked = true`. See [working with deprecated and revoked objects](#Working-with-deprecated-and-revoked-objects) for more information on revoked objects. |
 
-### Get all Techniques
-Once the stix2 Python library is installed and the corpus is acquired, we need to open the DataStore for querying:
+Note that because groups use software and software uses techniques, groups can be considered indirect users of techniques used by their software. See [getting techniques used by a group's software](#Getting-techniques-used-by-a-groups-software), below.
+
+# Accessing ATT&CK Data in Python
+There are several ways to acquire the ATT&CK data in Python. All of them will provide an object 
+implementing the DataStore API and can be used interchangeably with the recipes provided in the [recipes](#Python-Recipes) section.
```

Comment:
This leaves out any mention of the Python library--that it exists, how to get it, etc. That would be valuable for users who aren't familiar with it.

Reply:
Excellent point. We don't have a `requirements` file or anything so this is definitely something we should mention.

Reply:
I've added blurbs pointing to the library and documentation in the _Introduction_, _Accessing ATT&CK Data in Python_ and _Python Recipes_ sections.